src/crimson/os/seastore/lba_manager/btree/lba_btree_node.h

```diff
@@ -171,6 +171,8 @@ struct LBALeafNode
 
     for (auto i : *this) {
       auto child = (LogicalCachedExtent*)this->children[i.get_offset()];
+      // Children may not be marked as stable yet,
+      // the specific order is undefined in the transaction prepare record phase.
```

Comment:
Is it also good to rename validate_stable_children() to validate_(stable_)children_upon_preparing() ?